### ISSUE TITLE
Filtering the Problems pane

### ISSUE THREAD

**Issue by DaveInCaz** [twinbasic/twinbasic#1594]
**Is your feature request related to a problem? Please describe.**
After loading VB6 projects into tB (beta 340) I tend to get hundreds of entries in the Problems pane. Many warnings, some errors. The length makes it inefficient to work with.

**Describe the solution you'd like**
It would be convenient if we could filter that list by type (error vs. warning) and by module. To some extent the latter can be done by opening/collapsing nodes but this is a bit slow to do by hand. (A collapse-all option would help).

An option to filtering to "current module" might be the way to go.

I like the spare nature of most of the tB IDE GUI and I'm reluctant to recommend adding clutter, but in this case it seems like some added controls for filtering would be helpful.

**Comment by WaynePhillipsEA** [comment]
Some improvements in this area in BETA 346.

**Comment by DaveInCaz** [comment]
Indeed, the error/warning count and filter works great in 346, and also the collapse/expand options in the context menu.

One suggestion is that when I hid all warnings, most of the nodes become empty but remain in the treeview. I wonder if it would be better to hide nodes which have no visible children, or maybe put a count or some other indicator in their caption?
